### PR TITLE
Add  OVDMX device support

### PR DESCRIPTION
**Pull request by peternewman**
Hi @janovesaltvedt ,

I see you've added support for your ovdmx dongle to OLA and for the dongle here:
https://github.com/JanOveSaltvedt/ovdmx

We can get this merged into OLA mainline if you'd like as you seem to have a little bit of a tree of users of your codebase:
https://github.com/JanOveSaltvedt/ola-ovdmx/network/members

From a quick look the initial code doesn't look too far off being ready to merge.

### CODE REVIEW

**Comment by peternewman** [comment]
This commit probably wants adding too: https://github.com/JanOveSaltvedt/ola-ovdmx/commit/5b959726b8fd7de7adc1d76692fff3e9b81448ed

@aasmundeldhuset are you interested in helping to get this merged too?

**Comment by peternewman** [comment]
Or @stemnic

**Comment by stemnic** [comment]
I think it would be nice to have ovdmx integrated with the main branch of OLA.
As @JanOveSaltvedt said the pull should be set to 0, 0 which currently matches the default firmware. However, I think the plugin needs some work though before it can be merged with OLA since I don't think it currently compiles with the latest version without errors

File: plugins/ovdmx/OVDmxPlugin.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class OVDmxPlugin: public Plugin {
+ public:
+    explicit OVDmxPlugin(PluginAdaptor *plugin_adaptor):
+      Plugin(plugin_adaptor) {
+    }
+
+    std::string Name() const { return PLUGIN_NAME; }
+    std::string Description() const;
+    ola_plugin_id Id() const { return OLA_PLUGIN_EXPERIMENTAL; }
```
Comment (peternewman) [inline plugins/ovdmx/OVDmxPlugin.h]: See the notes here for how plugin IDs get assigned now: https://github.com/OpenLightingProject/ola/blob/master/common/protocol/Ola.proto#L72-L78

**Comment by peternewman** [comment]
The initial compile failure on Linux is fairly simple @stemnic , just see the notes here:
https://github.com/OpenLightingProject/ola/blob/master/common/protocol/Ola.proto#L72-L78

**Comment by stemnic** [comment]
Great I will take a look at it and see if I can make it work

**Comment by peternewman** [comment]
> Great I will take a look at it and see if I can make it work

Brilliant thanks. I think it's probably better if you re-open the PR because then I can approve it for merge (and you are a LOT more involved in the codebase than me, so I'll close this one so it lets you open a fresh one; at least I think it will then).

**Pull request closed without merging**